apps/www/content/docs/components/combobox.mdx

```diff
@@ -81,6 +82,7 @@ export function ComboboxDemo() {
         <Command>
           <CommandInput placeholder="Search framework..." />
           <CommandEmpty>No framework found.</CommandEmpty>
+          <CommandList>
```

Comment:
Also faced this issue but this repo have `CommandGroup` unwrapped in `CommandList` in several places ([here for example](https://github.com/shadcn-ui/ui/blob/bebc2843f0e0daa4e508def74fb3ba8a08e98f6f/apps/www/app/(app)/examples/playground/components/preset-selector.tsx#L52)). It works with `cmdk` version `0.2.0` though but not with `1.0.0`.